brokers/redis/src/options.rs

```diff
@@ -3,10 +3,10 @@ use structopt::StructOpt;
 #[derive(Debug, StructOpt)]
 #[structopt(name = "spectacles-redis")]
 pub struct Opt {
-	#[structopt(long, short, default_value = "localhost:6379")]
+	#[structopt(long, short, default_value = "localhost:6379", env = "REDIS_ADDRESS")]
 	pub address: String,
-	#[structopt(long, short)]
+	#[structopt(long, short, env = "REDIS_GROUP")]
 	pub group: String,
-	#[structopt(long, short)]
+	#[structopt(long, short, env = "REDIS_EVENTS")]
```

Comment:
Do you know how to specify a Vec using environment variables? The structopt documentation doesn't describe this behavior.

Reply:
Failed to notice this, seems like structopt actually doesn't support it. However config-rs supports this with `.list_separator(" ")`, so probably a good idea to use that

Reply:
Using `config` instead of `structopt` to allow environment variables and file-based configs